src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy

```diff
@@ -571,7 +571,8 @@ public abstract class GenerateProtoTask extends DefaultTask {
   @Internal
   @PackageScope
   Collection<File> getOutputSourceDirectories() {
-    Collection<File> srcDirs = []
+    // insertion point requires the same output source directories as the java plugin. Using a set to avoid duplication.
+    Collection<File> srcDirs = new HashSet<>()
```

Comment:
some interesting linter issue, if i correctly remember, linter expects to be `[] as Set<File>`